optbinning/scorecard/plots.py

```diff
@@ -0,0 +1,289 @@
+"""
+Modules with plots for visualizing model performance.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.metrics import roc_curve, roc_auc_score
```

Comment:
Add space between import and from import.

Reply:
Done.